### ISSUE TITLE
heroku_san should support https://git.heroku.com/ repo URIs as a configurable option

### ISSUE THREAD

**Issue by jikamens** [fastestforward/heroku_san#173]
Heroku seems to be encouraging people to use https://git.heroku.com/ repo URIs rather than git@github.com: URIs, i.e., https instead of ssh. Heroku\_san should at least allow this as an option, if not make it the default.

Note that this would fix #165, a long-standing bug which really deserves a better fix than a workaround buried in the comments of an open issue. Since Heroku now supports and recommends the accounts plugin to the toolbelt, heroku\_san should be able to deal with it.

**Comment by kmayer** [comment]
I agree! I've been unable to work on this fix for months, but maybe today
is the day. I have some time off :-)

On Mon, Dec 28, 2015 at 10:44 AM, Jonathan Kamens <notifications@github.com>
wrote:

> Heroku seems to be encouraging people to use https://git.heroku.com/ repo
> URIs rather than git@github.com: URIs, i.e., https instead of ssh.
> Heroku_san should at least allow this as an option, if not make it the
> default.
>
> Note that this would fix #165
> <https://github.com/fastestforward/heroku_san/issues/165>, a
> long-standing bug which really deserves a better fix than a workaround
> buried in the comments of an open issue. Since Heroku now supports and
> recommends the accounts plugin to the toolbelt, heroku_san should be able
> to deal with it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fastestforward/heroku_san/issues/173>.
>



-- 
ken@bitwrangler.com | (808) 722-6142 (c)

**Comment by kmayer** [comment]
A breaking test would be helpful.
